daemon/mgr/container.go

```diff
@@ -165,6 +169,12 @@ type ContainerMgr interface {
 
 	// Commit commits an image from a container.
 	Commit(ctx context.Context, name string, options *types.ContainerCommitOptions) (*types.ContainerCommitResp, error)
+
+	ContainerStatPath(ctx context.Context, name string, path string) (stat *types.ContainerPathStat, err error)
```

Comment:
could you add the comment for functions?

Reply:
sure